rust_src/admin/remacs-helpers.el

```diff
@@ -0,0 +1,34 @@
+;;; Package -- helpers for Remacs porting
+
+(defun remacs-helpers/make-rust-args-from-C-worker (input)
+  "Transform function C arguments INPUT into Rust style arguments."
+  (mapconcat (lambda (arg) (let* ((is-struct (lambda (s) (string= s "struct")))
+                                  (pieces (cl-remove-if is-struct (split-string (string-trim arg) " ")))
+                                  (name (s-append ":" (car (last pieces))))
+                                  (rest (butlast pieces)))
+                             (if (s-starts-with? "\*" name)
+                                 (s-join " " (cons (s-chop-prefix "*" name)
+                                                   (cons "*" rest)))
```

Comment:
`*` -> `*mut`?

Reply:
I am ok with defaulting to mut or const.